loops-arrays-objects.js

```diff
@@ -32,7 +34,19 @@ const scores = [
 // D
 // ==========================================
 
-
+for (let i = 0; i < scores.length ; i++) {
+    if (scores[i].score < 60){
+        console.log("F")
+    } else if ((scores[i].score <70) && (scores[i].score > 60)){
```

Comment:
De `&& (scores[i] > 60)` hoef je niet te doen, omdat je de eerste if al false heeft opgeleverd, dus dan moet de score wel groter zijn dan 60
```suggestion
    } else if ((scores[i].score <70)){
```

Dit geldt natuurlijk ook voor alle else-if's hieronder